### ISSUE TITLE
Orbot does not change IP

### ISSUE THREAD

**Issue by michas0602** [guardianproject/orbot#427]
Strange issue:
I have two different mobiles with the same OS installed (Lineage 17.1, both up to date). On mobile one everything is fine.

On mobile two not - Orbot seems to work fine - but websites as "whatismyip" show me my normal IP? 

Very unsafe, as Orbot seems to work fine...no problem visible, same options as on mobile 1 are set!

I already checked OS' settings, VPN under connections shows Orbot and is active (same as on mobile 1).I also tried reinstall of Orbot without success.
Whats going on with my second mobile? Why does everything look ok but my IP is not changed? I am not sure if its a problem of Lineage or Orbot - what can I do now to find out whats going on here?

Version is 16.3.3 Rc1

**Comment by bitmold** [comment]
Are you certain that within Orbot the VPN settings are configured in the same way on both devices? With Android, a VPN Service can be on (you see the key in the status bar) but you can set/unset certain apps within Orbot to use the VPN

**Comment by michas0602** [comment]
Yes I am...its a 'klte'-device and I assume its a kernel problem somehow as I read there are some others having the same issue with the same phone

**Comment by ValZapod** [comment]
> websites as "whatismyip" show me my normal IP?

Then ot DOES NOT work.
